package.json

```diff
@@ -41,6 +41,7 @@
     "ember-export-application-global": "^1.0.5",
     "ember-font-awesome": "3.0.4",
     "ember-load-initializers": "^0.6.3",
+    "ember-pdf-js": "0.2.0",
```

Comment:
do we still need this if it's dropped now?

Reply:
we still need it... it does a bunch of the importing of the library and setup... were' just not using the actual supplied component.